cyclopts/help.py

```diff
@@ -190,16 +190,18 @@ def format_doc(root_app, app: "App", format: str = "restructuredtext"):
 
 
 def _get_choices(type_: Type) -> str:
+    choices: str = ""
     if get_origin(type_) is Union:
```

Comment:
not for this PR, but I'm willing to bet that this needs a similar fix for python3.10 pipe unions as in #80